### ISSUE TITLE
Video

### ISSUE THREAD

**Issue by andyjones81** [DFE-Digital/dfe-frontend#9]

**Comment by SteveMilnes** [comment]
Get help buying for schools are looking at video content. Worth talking to them about this when we pick this up.

**Comment by titlescreen** [comment]
Yeah we are wanting to embed video from YouTube as a task that is completed within the service. Users would need to watch the video as its a legal requirement, so we may have that as an extra that could not be needed by other teams.
